Backend/app/dashboard/provider-dashboard/src/API/ProviderPlatform/DynamicOfferDriver/Issue.hs

```diff
@@ -37,7 +41,7 @@ import "lib-dashboard" Tools.Auth hiding (BECKN_TRANSPORT)
 import "lib-dashboard" Tools.Auth.Merchant
 
 type API =
-  "issue"
+  "issueV2"
```

Comment:
not backward compatible, why to change only?

Reply:
added issueV2 for customer but kept it same for driver.